ignite/services/plugin/template/main.go.plush

```diff
@@ -2,117 +2,45 @@ package main
 
 import (
 	"context"
-	"fmt"
-	"path/filepath"
+	"os"
 
 	hplugin "github.com/hashicorp/go-plugin"
 
-	"github.com/ignite/cli/v28/ignite/services/chain"
 	"github.com/ignite/cli/v28/ignite/services/plugin"
+	"<%= ModuleName %>/cmd"
 )
 
 type app struct{}
 
-func (app) Manifest(ctx context.Context) (*plugin.Manifest, error) {
-	return &plugin.Manifest{
-		Name: "<%= Name %>",
-		// TODO: Add commands here
-		Commands: []*plugin.Command{
-			// Example of a command
-			{
-				Use:   "<%= Name %>",
-				Short: "Explain what the command is doing...",
-				Long:  "Long description goes here...",
-				Flags: []*plugin.Flag{
-					{Name: "my-flag", Type: plugin.FlagTypeString, Usage: "my flag description"},
-				},
-				PlaceCommandUnder: "ignite",
-				// Examples of adding subcommands:
-				// Commands: []*plugin.Command{
-				// 	{Use: "add"},
-				// 	{Use: "list"},
-				// 	{Use: "delete"},
-				// },
-			},
-		},
-		// TODO: Add hooks here
-		Hooks: []*plugin.Hook{},
-		SharedHost: <%= SharedHost %>,
-	}, nil
-}
-
-func (app) Execute(ctx context.Context, cmd *plugin.ExecutedCommand, api plugin.ClientAPI) error {
-	// TODO: write command execution here
-	fmt.Printf("Hello I'm the example-plugin plugin\n")
-	fmt.Printf("My executed command: %q\n", cmd.Path)
-	fmt.Printf("My args: %v\n", cmd.Args)
-
-	flags, err := cmd.NewFlags()
-	if err != nil {
-		return err
+func (app) Manifest(_ context.Context) (*plugin.Manifest, error) {
+	m := &plugin.Manifest{
+		Name: "<%= Name %>",<%= if (SharedHost) { %>
+		SharedHost: true,<% } %>
 	}
-
-	myFlag, _ := flags.GetString("my-flag")
-	fmt.Printf("My flags: my-flag=%q\n", myFlag)
-	fmt.Printf("My config parameters: %v\n", cmd.With)
-
-	// This is how the plugin can access the chain:
-	// c, err := getChain(cmd)
-	// if err != nil {
-	// 	return err
-	// }
-
-	// According to the number of declared commands, you may need a switch:
-	/*
-		switch cmd.Use {
-		case "add":
-			fmt.Println("Adding stuff...")
-		case "list":
-			fmt.Println("Listing stuff...")
-		case "delete":
-			fmt.Println("Deleting stuff...")
-		}
-	*/
-	
-	// ClientAPI call example
-	fmt.Println(api.GetChainInfo(ctx))
-	
-	return nil
+	m.ImportCobraCommand(cmd.New<%= Title %>(), "ignite")
```

Comment:
IMHO. we should incentivize the developers to use the manifest instead of the Cobra command. This method was created as a workaround so we could migrate the network commands to a plugin

Reply:
OK this is an interesting subject to discuss! Although for now at least using the cobra command is the only way we write our apps, and I don't think we have any example where we use the `Manifest` directly. 
@jeronimoalbi Any thoughts on this?

Reply:
cc @jeronimoalbi

Reply:
What @Pantani mentioned is right, I think Cobra was used just because the network app was originally part of the CLI, so the way it was migrated to an app was by keeping the original Cobra functionality.

Both implementation are valid but I think the Manifest should be the recommended way users should define commands and hooks.

Reply:
Do we have any example, documentation or standard template for Manifest without the `ImportCobraCommand` usage?

Reply:
> Do we have any example, documentation or standard template for Manifest without the `ImportCobraCommand` usage?

Interesting, all out apps are using the Cobra approach provably because they came after the network one so they are based on it. The idea of the original plugins is not to rely on Cobra if I recall correctly. Commands and Hooks can be defined in the Manifest directly and by doing so they would have less dependencies.

Reply:
Pushed a quick PR ignite/apps#38 with a way to use the manifest instead of depending on Cobra